src/Symfony/Component/Validator/Constraints/AbstractComparison.php

```diff
@@ -28,11 +29,20 @@ abstract class AbstractComparison extends Constraint
     public mixed $value = null;
     public ?string $propertyPath = null;
 
-    public function __construct(mixed $value = null, ?string $propertyPath = null, ?string $message = null, ?array $groups = null, mixed $payload = null, array $options = [])
+    #[HasNamedArguments]
+    public function __construct(mixed $value = null, ?string $propertyPath = null, ?string $message = null, ?array $groups = null, mixed $payload = null, ?array $options = null)
```

Comment:
`$value` and `$propertyPath` could benefit from CPP I think? We would then have on parameter per line, making the whole thing more readable

Reply:
I don't think this works right now because of the handling of the options array in the parent constructor (could be changed in 8.0 probably when we drop support for it).